ouroboros-network-framework/testlib/TestLib/InboundGovernor.hs

```diff
@@ -133,10 +133,13 @@ allValidRemoteTransitionsNames =
 -- Assuming all transitions in the transition list are valid, we only need to
 -- look at the 'toState' of the current transition and the 'fromState' of the
 -- next transition.
-verifyRemoteTransitionOrder :: [RemoteTransition]
+verifyRemoteTransitionOrder :: Bool -- ^ Check last transition: useful for
+                                    --    distinguish Diffusion layer tests
+                                    --    vs non-Diffusion ones.
```

Comment:
Why do we need this for diffusion tests?

Reply:
Because in Diffusion we truncate the log, so one can not really be sure if a broken chain of remote transitions ends in the correct final state or not.